reconcile/openshift_saas_deploy.py

```diff
@@ -120,6 +131,9 @@ def run(
     # - this is not a dry run
     # - there is a single saas file deployed
     notify = not dry_run and len(saas_files) == 1
+    skip_successful_deploy_notifications = False
+    if len(saas_files) > 0:
+        skip_successful_deploy_notifications = saas_files[0].skip_successful_deploy_notifications
```

Comment:
What would happen when `skip_successful_deploy_notifications` is not set in the saas file?

Reply:
The codebase, here and elsewhere, is expected to behave as before this PR. Semantically, it means that integrations need to opt into the skip notification.

Reply:
But `skip_successful_deploy_notifications ` is added by the schema file right? Have you try dry run the integration locally against a saas file that doesn't have this field set?

In genearl, I encourage adding the manual tests you have done locally to validate your change in the MR description.